[BUG] Steam store page's extra links are not displayed

### Expected Behavior
The link buttons to SteamDB, PCGamingWiki, etc are displayed on an individual app page around the right side: 
"Popular user-defined tags for this product: Tags

View Your Queue   

// Extra links list HERE //

Is this game relevant to you?"


### Actual Behavior
Screenshot: 
![image](https://user-images.githubusercontent.com/12727188/121440263-88bf2a00-c9c2-11eb-99ec-883b246b07bd.png)

The link buttons are not displayed with this console message: 
 Augmented Steam v2.0.1  https://augmentedsteam.com
app.js:1 V
app.js:1 Error while applying feature V
(anonymous) @ app.js:1
app.js:1 DOMException: Failed to execute 'insertAdjacentElement' on 'Element': The new child element contains the parent.
    at V._moveExtraLinks (chrome-extension://dnhpnfgdlenaccegplpojghhmaamnnfp/js/store/app.js:1:67738)
    at V.apply (chrome-extension://dnhpnfgdlenaccegplpojghhmaamnnfp/js/store/app.js:1:65229)
    at chrome-extension://dnhpnfgdlenaccegplpojghhmaamnnfp/js/store/app.js:1:136695
(anonymous) @ app.js:1


### Steps to Reproduce the Problem

  1. Open any individual app's Steam store page on Google Chrome
  2. Check the corner where extra links were used to be

### Specifications

  - Augmented Steam Version: v2.0.1
  - Browser Version : Google Chrome Version 91.0.4472.77 (Official Build) (64-bit)

--- a/src/js/Content/Features/Store/Common/FExtraLinks.js
+++ b/src/js/Content/Features/Store/Common/FExtraLinks.js
@@ -1,8 +1,5 @@
+import {HTML, HTMLParser, Localization, SyncedStorage} from "../../../../modulesCore";
 import {ContextType, Feature} from "../../../modulesContent";
-
-import {HTML} from "../../../../Core/Html/Html";
-import {Localization} from "../../../../Core/Localization/Localization";
-import {SyncedStorage} from "../../../../Core/Storage/SyncedStorage";
 
 export default class FExtraLinks extends Feature {
 
@@ -45,11 +42,13 @@ export default class FExtraLinks extends Feature {
 
             this._moveExtraLinks();
 
+            const appName = HTMLParser.clearSpecialSymbols(this.context.appName);
+
             if (SyncedStorage.get("showyoutube")) {
                 HTML.afterBegin(this._node,
                     this._getRightColLinkHtml(
                         "youtube_btn",
-                        `https://www.youtube.com/results?search_query=${encodeURIComponent(this.context.appName)}`,
+                        `https://www.youtube.com/results?search_query=${encodeURIComponent(appName)}`,
                         Localization.str.view_on_website.replace("__website__", "YouTube")
                     ));
             }
@@ -58,12 +57,10 @@ export default class FExtraLinks extends Feature {
                 HTML.afterBegin(this._node,
                     this._getRightColLinkHtml(
                         "twitch_btn",
-                        `https://www.twitch.tv/directory/game/${encodeURIComponent(this.context.appName.replace(/(\u2122)/g, "")
-                            .replace(/(\xAE)/g, ""))}`,
+                        `https://www.twitch.tv/directory/game/${encodeURIComponent(appName)}`,
                         Localization.str.view_on_website.replace("__website__", "Twitch")
                     ));
             }
-
 
             if (SyncedStorage.get("showpcgw")) {
                 HTML.afterBegin(this._node,

--- a/src/js/Content/Features/Store/Common/FExtraLinks.js
+++ b/src/js/Content/Features/Store/Common/FExtraLinks.js
@@ -9,7 +9,7 @@ export default class FExtraLinks extends Feature {
         if (context.type === ContextType.APP) {
             this._type = "app";
             this._gameid = context.appid;
-            this._node = document.querySelector("#ReportAppBtn").parentNode;
+            this._node = document.querySelector("#shareEmbedRow");
         } else if (context.type === ContextType.SUB) {
             this._type = "sub";
             this._gameid = context.subid;
@@ -137,7 +137,7 @@ export default class FExtraLinks extends Feature {
 
     _moveExtraLinks() {
 
-        const usefulLinks = this._node.parentNode;
+        const usefulLinks = this._node;
         usefulLinks.classList.add("es_useful_link");
 
         const sideDetails = document.querySelector(".es_side_details_wrap");

